helix-syntax/Cargo.toml

```diff
@@ -4,8 +4,11 @@ version = "0.2.0"
 authors = ["Blaž Hrastnik <blaz@mxxn.io>"]
 edition = "2018"
 license = "MPL-2.0"
-
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+description = "Tree-sitter grammars support"
+categories = ["editor"]
+repository = "https://github.com/helix-editor/helix"
+homepage = "https://helix-editor.com"
+include = ["src/**/*", "languages/**/*", "build.rs", "!**/docs/**/*", "!**/test/**/*", "!**/examples/**/*", "!**/build/**/*"]
```

Comment:
I don't think we need to manually specify `include` right?

Reply:
This is unrelated to docs.rs but it fact cargo _will_ package more than strictly necessary yes.
See https://github.com/the-lean-crate/cargo-diet